.github/workflows/main.yml

```diff
@@ -1,6 +1,6 @@
 name: 'CI: Pull request'
 
-on: push
+on: pull_request_target
```

Comment:
could we also add `push` in addition to `pull_request_target`?

Reply:
Done!